### ISSUE TITLE
Is there a way to see files in recylebin from filemanager?

### ISSUE THREAD

**Issue by kyoyacchi** [SimpleMobileTools/Simple-Gallery#2450]
Like i said in title.

I want to update the app but i don't want recylebin to disappear.

**Comment by tibbi** [comment]
nope, the bin is stored inside the gallery only

**Comment by OkyDooky** [comment]
@kyoyacchi 
I don't think that will delete the contents of the Recycle Bin. But I've been using the F-Droid version, so it could be different for the PlayStore version. If that's a concern for you, I recommend you either restore the contents or use the "Copy To" option to group them into a unique/specific folder for later. That other folder can be accessed by any file manager.

**Comment by kyoyacchi** [comment]
> @kyoyacchi I don't think that will delete the contents of the Recycle Bin. But I've been using the F-Droid version, so it could be different for the PlayStore version. If that's a concern for you, I recommend you either restore the contents or use the "Copy To" option to group them into a unique/specific folder for later. That other folder can be accessed by any file manager.

Yes, i did same thing before updating app, and yes the recyle bin is gone after the update. I'll check froid version too

**Comment by OkyDooky** [comment]
@kyoyacchi Huh. By "gone", do you mean that it never comes back? Double check the Settings section, since there are multiple toggleable (check mark) options for enabling and showing the Recycle Bin. It could be that those got somehow reset or disabled after the update (it does sound like you're jumping from a really old version to a really new version, so something like that might be more likely to occur).

Did you buy the PlayStore version (as in pay money)? If you did, you may want to stick with that, since it will have the new image (and video!) editor. The F-Droid version, which I use, is completely free (and free from closed-source code), but that's it's only advantage.
